recipes/gstreamer/all/conandata.yml

```diff
@@ -1,4 +1,10 @@
 sources:
+  "1.22.3":
+    url: "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.22.3.tar.xz"
+    sha256: "9ffeab95053f9f6995eb3b3da225e88f21c129cd60da002d3f795db70d6d5974"
+  "1.20.6":
+    url: "https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-1.20.6.tar.xz"
```

Comment:
```suggestion
    url: "https://gitlab.freedesktop.org/gstreamer/gstreamer/-/archive/1.20.6/gstreamer-1.20.6.tar.gz"
```

Reply:
GitLab/GitHub auto-generated archives cannot be trusted.